src/main/helm/helm-service/get-helm-release-resources/get-helm-release-resources.test.ts

```diff
@@ -65,150 +65,9 @@ describe("get helm release resources", () => {
 
       const actual = await actualPromise;
 
-      expect(actual).toEqual([]);
-    });
-
-    describe("when call for manifest resolves", () => {
-      beforeEach(async () => {
-        await execHelmMock.resolve({
-          callWasSuccessful: true,
-          response: `---
-apiVersion: v1
-kind: SomeKind
-metadata:
-  name: some-resource-with-same-namespace
-  namespace: some-namespace
----
-apiVersion: v1
-kind: SomeKind
-metadata:
-  name: some-resource-without-namespace
----
-apiVersion: v1
-kind: SomeKind
-metadata:
-  name: some-resource-with-different-namespace
-  namespace: some-other-namespace
----
-`,
-        });
-      });
-
-      it("calls for resources from each namespace separately using the manifest as input", () => {
-        expect(execFileWithStreamInputMock.mock.calls).toEqual([
-          [
-            {
-              filePath: "/some-kubectl-path",
-              commandArguments: ["get", "--kubeconfig", "/some-kubeconfig-path", "-f", "-", "--namespace", "some-namespace", "--output", "json"],
-              input: `---
-apiVersion: v1
-kind: SomeKind
-metadata:
-  name: some-resource-with-same-namespace
-  namespace: some-namespace
----
-apiVersion: v1
-kind: SomeKind
-metadata:
-  name: some-resource-without-namespace
----
-`,
-            },
-          ],
-
-          [
-            {
-              filePath: "/some-kubectl-path",
-              commandArguments: ["get", "--kubeconfig", "/some-kubeconfig-path", "-f", "-", "--namespace", "some-other-namespace", "--output", "json"],
-              input: `---
-apiVersion: v1
-kind: SomeKind
-metadata:
-  name: some-resource-with-different-namespace
-  namespace: some-other-namespace
----
-`,
-            },
-          ],
-        ]);
-      });
-
-      it("when all calls for resources resolve, resolves with combined result", async () => {
-        await execFileWithStreamInputMock.resolveSpecific(
-          ([{ commandArguments }]) =>
-            commandArguments.includes("some-namespace"),
-          {
-            callWasSuccessful: true,
-
-            response: JSON.stringify({
-              items: [{ some: "item" }],
-
-              kind: "List",
-
-              metadata: {
-                resourceVersion: "",
-                selfLink: "",
-              },
-            }),
-          },
-        );
-
-        await execFileWithStreamInputMock.resolveSpecific(
-          ([{ commandArguments }]) =>
-            commandArguments.includes("some-other-namespace"),
-          {
-            callWasSuccessful: true,
-
-            response: JSON.stringify({
-              items: [{ some: "other-item" }],
-
-              kind: "List",
-
-              metadata: {
-                resourceVersion: "",
-                selfLink: "",
-              },
-            }),
-          },
-        );
-
-        const actual = await actualPromise;
-
-        expect(actual).toEqual([{ some: "item" }, { some: "other-item" }]);
```

Comment:
About the missing tests: I think there is no new substitute for this removed test.